Dockerfile

```diff
@@ -56,13 +56,12 @@ ONBUILD ADD cpanfile aptfile /app/
 # go through the CPAN deps. Once those are all done, remove anything
 # that we would have pulled in as a build dep (compilers, for example)
 # unless they happened to be in the aptfile.
-ONBUILD RUN if [ -s /app/aptfile ]; then \
+ONBUILD RUN if [ -r /app/aptfile ]; then \
     apt-get -y -q update \
     && apt-get -y -q --no-install-recommends install $(cat /app/aptfile); \
   fi \
-  && cpanm --notest --quiet --installdeps --with-recommends . \
+  && /usr/bin/dumb-init -- cpanm --notest --quiet --installdeps --with-recommends . \
```

Comment:
Why specifically `cpanm` requiring this? If it's a problem there, it'd be a problem *everywhere*, even apt install tends to run Perl code.

Reply:
I'm not sure if its even a Perl-only problem, as the case for this stems from cpanm subprocs like make/gcc/cc1 becoming zombies for some reason (hinting that some sub-parent is not handling signals,) and furthermore it doesn't happen on all cpanm installs.  I agree that this might be placed further up in this RUN for the best coverage.